include/fastrtps/utils/collections/foonathan_memory_helpers.hpp

```diff
@@ -49,12 +50,9 @@ std::size_t memory_pool_block_size(
         num_elems = 1u;
     }
 
-    // Make room for debug fence
-    num_elems += 2u;
-
-    return node_size * num_elems       // Room for elements
-        + MemoryPool::min_node_size    // Room for free_list nodes
-        + 16u;                         // Additional fence space
+    return num_elems * (node_size                                                  // Room for elements
```

Comment:
Should we take into account the `memory_pool::min_node_size` ?

Reply:
Done